src/main/java/cat/udl/eps/softarch/myroutes/domain/RouteFollowed.java

```diff
@@ -40,7 +40,6 @@ public class RouteFollowed extends UriEntity<Long> {
 
     @JsonProperty(access = JsonProperty.Access.READ_ONLY)
     @NotNull
-    @ManyToOne
-    public Route routeOrigin;
+    public String follows;
```

Comment:
No need to change the type in the backend, please, keep it a Route with the updated name:
```
@ManyToOne
public Route follows;
```

In the frontend you can also store the Route entity, but when sending to the backend, if the entity already exists, use its URI instead (which is stored as a String).

Reply:
Like it is done for RouteVersion:
https://github.com/UdL-EPS-SoftArch/myroutes/pull/32/commits/33201e01bb3c91c7c0f469944f046026abb03247

Reply:
All the changes are already submitted in both front and backend.